### ISSUE TITLE
Use of alias inside Struct block

### ISSUE THREAD

**Issue by tiagoefmoraes** [bbatsov/rubocop#5746]
Style/Alias offense registered in `Struct` block. But ruby doesn't suport `alias_method` in it.

--------
## Expected behavior

no offense detected

## Actual behavior

`Style/Alias: Use alias_method instead of alias.` is reported

## Steps to reproduce the problem

In a file with:
```ruby
Struct.new(:text) do
  alias full_text text
end
```
run rubocop

But if alias_method is used as suggested:

```ruby
Struct.new(:text) do
  alias_method full_text text
end
```
ruby now gives this error:
```
Failure/Error: alias_method full_text text

NameError:
  undefined local variable or method `text' for #<Class:#<Class:0xaf78c87>>
```
## RuboCop version

```
$ rubocop -V
0.54.0 (using Parser 2.5.0.4, running on ruby 2.4.3 x86_64-linux)
```

**Comment by Drenmi** [comment]
`#alias_method`, unlike `alias`, is not a keyword, but a regular method. Thus you need to use method call syntax:

```ruby
alias_method :full_text, :text
```

**Comment by tiagoefmoraes** [comment]
Sorry for that, it's really embarrassing.

Thank you

**Comment by Drenmi** [comment]
No problem. I get the order of the arguments to `alias` wrong all the time. 🙂
